source/extensions.h

```diff
@@ -18,14 +18,14 @@
 #include <string>
 
 #include "enum_set.h"
-#include "message.h"
-#include "spirv-tools/libspirv.hpp"
+#include "spirv-tools/libspirv.h"
 
 namespace libspirv {
 
 // The known SPIR-V extensions.
 // TODO(dneto): Consider auto-generating this list?
-// When updating this list, consider also updating ParseSpvExtensionFromString.
+// When updating this list, consider also updating ParseSpvExtensionFromString
```

Comment:
It's becoming even more important to automatically parse extensions from the JSON grammar file ...